lib/AST/NameLookup.cpp

```diff
@@ -2758,6 +2758,21 @@ void FindLocalVal::checkPattern(const Pattern *Pat, DeclVisibilityKind Reason) {
   
 void FindLocalVal::checkParameterList(const ParameterList *params) {
   for (auto param : *params) {
+    if (param->hasAttachedPropertyWrapper()) {
+      // FIXME: This shouldn't be needed, but only this set the interface type
+      // of the auxiliary variables.
+      (void)param->getPropertyWrapperBackingPropertyType();
+
+      auto vars = param->getPropertyWrapperAuxiliaryVariables();
+      checkValueDecl(vars.localWrappedValueVar,
+                     DeclVisibilityKind::FunctionParameter);
+      checkValueDecl(vars.projectionVar, DeclVisibilityKind::FunctionParameter);
```

Comment:
I think we need to check at least projectionVar for null.  Not sure about the other two -- they seem like they're intended to not be null, but I'm not sure what happens in invalid code.

Reply:
Yeah, I changed to check all of them, just in case.
Also, to support local variables, I moved the logic into `checkValueDecl()`.